generate_test_data.py

```diff
@@ -0,0 +1,32 @@
+from git import Repo
+import shutil
+from clinical_etl import CSVConvert
+import argparse
+import os
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description="A script that ingests clinical data into Katsu")
```

Comment:
description copied from somewhere else? should be about generating test data?

Reply:
Have updated this in latest commits